aws_cloudformation_policies/aws_cloudformation_stack_drifted.py

```diff
@@ -1,2 +1,19 @@
+# CloudFormation stacks tagged with "STACK=(name)" will be marked as passing.
+IGNORE_STACK_TAGS = {
+    'panther-bootstrap-gateway', 'panther-cloud-security', 'panther-core',
+    'panther-log-analysis'
+}
+
+
 def policy(resource):
-    return resource['DriftInformation']['StackDriftStatus'] != "DRIFTED"
+    if resource['DriftInformation']['StackDriftStatus'] != "DRIFTED":
+        return True
+
+    # Some of Panther's own stacks contain Lambda functions which will always show as "drifted."
+    # Panther stacks have a fixed "Stack" tag, even though the real stack name is dynamic.
+    tags = resource['Tags']
+    if tags.get('Application') == 'Panther' and tags.get(
```

Comment:
IIRC - the drift is because of SAM translation

Are all stacks affected?

Reply:
All of our stacks with Lambda source are affected, but that is not every Panther stack. That's why I just hardcoded the names of stacks known to drift. (I checked a live hosted deployment to confirm)